.github/workflows/release.yaml

```diff
@@ -5,46 +5,50 @@ on:
     - master
   schedule:
   - cron: "0 */8 * * *"
+  workflow_dispatch: {}
 jobs:
   release:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v1
     - name: Prepare checking latest kubectl version
-      run: docker build . -t line/kubectl-kustomize:kubectl-version --target latest-kubectl-version
+      env:
+        REPO_OWNER: ${{github.repository_owner}}
+      run: docker build . -t ${REPO_OWNER}/kubectl-kustomize:kubectl-version --target latest-kubectl-version
     - name: Check latest kubectl version
       id: kubectl
+      env:
+        REPO_OWNER: ${{github.repository_owner}}
       run: |
-        if version=$(docker run line/kubectl-kustomize:kubectl-version); then
+        if version=$(docker run ${REPO_OWNER}/kubectl-kustomize:kubectl-version); then
```

Comment:
Similarly, I added changes to make it easier for me to push to my docker hub for testing. Happy to reverse it here.

Reply:
It makes sense for using `REPO_OWNER`, but it might cause confuses for some cases. (e.g. using different username from GitHub)

I think it would be better for using hard-coded username for now.